src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const trs = table.rows;
+
+function addColumn(lines) {
+  const rows = [...lines];
+
+  rows.forEach((row) => {
+    const tr = row.cells;
+    const secondTr = tr[1].cloneNode(true);
```

Comment:
Cloning the second cell (`tr[1]`) might not be the intended behavior if the new column should contain different content. Consider whether this is the desired action.